msc_pygeoapi/loader/climate_archive.py

```diff
@@ -1184,13 +1195,17 @@ def add(
     else:
         datasets_to_process = [dataset]
 
+    # if no date, station or starting_from is provided, then it is
+    # a full reindexing
+    full_reindex = not (date or station or starting_from)
```

Comment:
`full_reindex` is to evaluate to `True`  if date, station and starting are all None, otherwise it is false.  This proposed change would evaluate to `False` if all three values are `None`.

Would this be more explicit?
```
full_reindex = all(
    value is None for value in [date, station, starting_from]
)
 ```

Reply:
We can also do:

```python
full_reindex = all([
    date is None,
    station is None,
    starting_from is None
])
```

Reply:
Excellent. Updated! Thanks for the feedback.